src/transformers/training_args.py

```diff
@@ -53,14 +53,16 @@
     torch_required,
 )
 
-
 if is_torch_available():
     import torch
     import torch.distributed as dist
 
 if is_torch_tpu_available(check_device=False):
     import torch_xla.core.xla_model as xm
-
+    # torchrun support
+    import torch_xla.distributed.xla_backend
+    if os.environ.get("WORLD_SIZE"):
+        torch.distributed.init_process_group(backend="xla")
```

Comment:
This should be moved to the `_setup_devices` method of `TrainingArguments`. It shouldn't be done on import.

Reply:
Thanks. Unfortunately I can't move it into _setup_devices for some reason as it run into device not found error. Will investigate.